backend/nginx/conf/nginx-docker.conf

```diff
@@ -35,6 +37,7 @@ http {
     server {
         listen       21011;
         server_name  localhost;
+        access_log   /dcef/cache/nginx/logs/access.log
```

Comment:
You may want to reroute error_log too. Also missing required ; at the end.

for example:
```conf
access_log   /dcef/cache/nginx/logs/access.log;
error_log    /dcef/cache/nginx/logs/error.log;
```

This change also need /dcef/cache/nginx/logs/ directory to exists, else nginx will complain

Reply:
Force pushed with your recommended change, Thanks! Also added a directory make line to `run_container.sh`.